go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
 	github.com/michaelquigley/pfxlog v0.0.0-20190813191113-2be43bd0dccc
 	github.com/mitchellh/mapstructure v1.1.2
+	github.com/netfoundry/secretstream v0.1.1
```

Comment:
Were there reviews for this?

Reply:
Yes, https://github.com/netfoundry/secretstream/pull/1